weeks/week07/week07.md

```diff
@@ -1,3 +1,91 @@
 # Week 7: Web Automation with Playwright
 
-Coming soon.
+## Playwright general information:
+Playwright is the powerful tool for Web Test automation (UI and API) in different programming languages:
+- TypeScript, 
+- JavaScript,
+- .NET,
+- Java.
+- Python
+
+It is 
+- Cross-browser,
+- Cross-platform,
+- Support mobile testing
+
+The scope of this page: Playwright - Python
+
+## Playwright - Python General information:
+1. Convenient for Web and API testing
+2. Intuitive integration with Pytest Test Automation Framework
+3. Intuitive integration with Allure
+4. Inherited  support for all Python functionality
+5. Easy to use within CI/CD (e.g. Jenkins)
+
+## References:
+
+1. [Playwright - Python oficial page](https://playwright.dev/python/)
+2. [Installation](https://playwright.dev/python/docs/intro)
+3. [Pytest fixtures](https://docs.pytest.org/en/6.2.x/fixture.html#autouse-fixtures-fixtures-you-don-t-have-to-request)
+
+## Installation:
+From Python point of view Playwright is just a package
+
+Installation: 
+```python
+pip install pytest-playwright
+```
+
+## Launching:
+From Python point of view Playwright is enhanced pytest
+
+Launching: 
+```python
+pytest
+```
+
+## Test:
+1. Action
+2. Assertion
+3. Using fixtures
+
+## UI testing
```

Comment:
It is important to talk about Page Object Model and Page Object Factory with examples. You can create a folder for a particular sample and add a few tests exploring PO there.